test/functionalTest/orionWS.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys
+import websocket
+
+msg = 0;
+
+def on_message(ws, message):
+    print message
+
+    global msg
+    msg -= 1
+    if (msg == 0):
+        ws.close()
+
+def on_error(ws, error):
+    print error
+
+def on_close(ws):
+    print "Connection closed"
+
+def on_open(ws):
+    ws.send(sys.argv[3])
+
+def show_help(reason):
```

Comment:
It would be great to include a small .test (under cases/1181_websockets) which shows how this orionWs works. Something like this:

```
...
--SHELL--
echo "00. Just to test"
echo "==========="
payload='{
  "verb" : "POST",
  "url" : "/v2/entities",
  "params" : {"options" : "keyValues"},
  "headers": {"Fiware-service": "my_service"},
  "payload" : {"type": "T", "id": "E1","temp": 1}
}'
orionWs localhost:9001 ngsiv2-json "$payload" 1
echo
echo
--REGEXEPECT--
<fill with the expected value>
```
The purpose is twofold:

1. Ensure that orionWs is working as expected, well integrated in the test harness suite
2. Have an example to guide the work of the team members that at the end will implement actual test harness for websockets.